templates/workspaces/base/terraform/variables.tf

```diff
@@ -48,6 +48,11 @@ variable "register_aad_application" {
   description = "Create an AAD application automatically for the Workspace."
 }
 
+variable "aad_redirect_uris_b64" {
```

Comment:
why do we accept a dictionary when the terraform just expects a comma seperated list of URIs? Is this a porter thing?

Reply:
we need to have a list of objects in the property bag as we need a way of identifying a URI to remove / replace - so it's a key/value pair with the key being something that doesn't change.